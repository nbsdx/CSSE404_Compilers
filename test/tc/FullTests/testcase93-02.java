class MainClass {
	public static void main(String[] args){
		
	}
}

class ChainRestaurant {
	boolean isCheap;
	boolean isHealthy;
	boolean hasCartoonishMascot;
	
	int locations;
	
	public int setLocations(int val){
		locations = val;
		return val;
	}
	
}

class TacoBell extends ChainRestaurant {
	public TacoBell setVars() {
		isCheap = true;
		isHealthy = false;
		hasCartoonishMascot = 3;
		
		int d = this.setLocations(5);
		return this;
	}
}