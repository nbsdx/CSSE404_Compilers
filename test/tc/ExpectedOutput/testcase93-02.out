Cannot assign type int to variable hasCartoonishMascot of type boolean
